### PR TITLE
Changed unit tests using unittest library

### PR DESCRIPTION
**Pull request by hgKang02**

### CODE REVIEW

File: syncode/runtime_eval.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from infer import Syncode
```
Comment (shubhamugare) [inline syncode/runtime_eval.py:1]: what is this file?

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -30,10 +30,4 @@ jobs:
           key: files-${{ hashFiles('syncode/parsers/grammars/python_grammar.lark', 'syncode/dfa_mask_store.py') }}
     - name: Run Tests
       run: |
-        python tests/test_grammar_python.py
-        python tests/test_grammar_go.py
-        python tests/test_calc.py
-        python tests/test_lr_parser.py
-        python tests/test_language_model.py
-        python tests/test_dfa_mask_python.py --independent
```
Comment (shubhamugare) [inline .github/workflows/run_tests.yml:38]: For `test_dfa_mask_python` we were only running tokenizer independent tests. Can you do something similar? Or maybe just stop running all of these tests.

File: tests/test_dfa_mask_python.py [hunk 3]
```diff
@@ -6,215 +7,291 @@
...
+    def test_dfa_mask11(self):
+        ac_list = self.dfa_mask.get_overapprox_tokens_mask(ParseResult({AcceptSequence(['STRING'])}, "'", RemainderState.INCOMPLETE, next_ac_indents=None), get_list=True)
+        self.assertIn(" '.", ac_list)
+
+
+import argparse
+import unittest
+
+if __name__ == '__main__':
+    # argparser = argparse.ArgumentParser(description='Run tests for DFAMaskStore object')
```
Comment (shubhamugare) [inline tests/test_dfa_mask_python.py:215]: You can remove uncommented code if not needed

File: tests/test_dfa_mask_python.py [hunk 4]
```diff
@@ -6,215 +7,291 @@
...
-        tests_codegen = [test_dfa_mask10, test_dfa_mask11]
-        common.run_tests(tests_codegen)
+        suite = unittest.TestLoader().loadTestsFromTestCase(TestDFAMaskWizard)
+        unittest.TextTestRunner().run(suite)
+    # unittest.main()
+
+
+
+
+# import argparse
```
Comment (shubhamugare) [inline tests/test_dfa_mask_python.py:237]: What's all this?

File: syncode/runtime_eval.py (hunk 1 above)
Comment (hgKang02) [reply syncode/runtime_eval.py:1]: This file is for me to just run the runtime experiment automatically. I forgot to not include this in gitignore.

File: .gitignore [hunk 5]
```diff
@@ -7,3 +7,4 @@ syncode/core/__pycache__
 tmp*
 cache/
 .ipynb_checkpoints/
+syncode/runtime_eval.py
```
Comment (shubhamugare) [inline .gitignore]: It'd be better if you move this to your personal .gitignore

**Pull request merged**